src/documentation/install.page

```diff
@@ -41,19 +41,19 @@ The only dependency of uniqush-push is [redis], which is a key-value database.
 
 ### Running uniqush-push
 
-- If the uniqush-push is installed from binary packages from the [download page], then there will be two files copied to your system:
+- If uniqush-push is installed via the binary packages from the [download page], then there will be two files copied to your system:
     * *uniqush-push*, which is an executable file, will be copied to `/usr/bin`.
```

Comment:
- `/usr/bin/uniqush-push`, which is an executable file
- `/etc/uniqush/uniqush-push", which is a configuration file